airflow/providers/openlineage/extractors/base.py

```diff
@@ -84,6 +84,12 @@ def get_operator_classnames(cls) -> list[str]:
     def extract(self) -> OperatorLineage | None:
         try:
             return self._get_openlineage_facets(self.operator.get_openlineage_facets_on_start)  # type: ignore
+        except ImportError:
+            self.log.error(
+                "OpenLineage provider method failed to import OpenLineage integration. "
+                "This should not happen."
```

Comment:
should we not add what user should do when this error occurs?

Reply:
Fixed.